### PR TITLE
docs: fix link to repo in demo-slides

### PR DESCRIPTION
**Pull request by lehors**
The current link still points to the old repo.

### CODE REVIEW

**Review by leseb — APPROVED** [review]

File: docs/demo-slides.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ source venv/bin/activate.fish
 Install CLI
 
 ```fish
-pip install git+https://github.com/instructlab/cli.git@stable
+pip install git+https://github.com/instructlab/instructlab.git@stable
```
Comment (bjhargrave) [inline docs/demo-slides.md:37]: Now that instructlab is in PyPI, shouldn't we recommend `pip install instructlab` instead?

Comment (leseb) [reply docs/demo-slides.md:37]: Agreed.

Comment (bjhargrave) [reply docs/demo-slides.md:37]: Also, we are deprecating the `stable` branch, so this reference also needs to be removed.

Comment (lehors) [reply docs/demo-slides.md]: Good point. Done.

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**